### PR TITLE
Fixed typos in comments

### PR DESCRIPTION
**Pull request by RomainSanchez**
I am targetting this branch, because it is BC (and pedantic).

### CODE REVIEW

File: Resources/public/Admin.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ var Admin = {
     collectionCounters: [],
 
     /**
-     * This function must called when a ajax call is done, to ensure
-     * retrieve html is properly setup
+     * This function must be called when an ajax call is done, to ensure
+     * retrieved html is properly setup
```
Comment (greg0ire) [inline Resources/public/Admin.js]: "the retrieved" ?

**Comment by OskarStark** [comment]
Thank you @RomainSanchez 

Btw, if it is `pedantic`, no need for the Changelog-Section in the PR header (I already removed that)

**Comment by RomainSanchez** [comment]
> we will squash your commits unless you know how to do it yourself (or want to learn)

I am not on my personal time so time is money.I'll let you do it this time if you don't mind

**Comment by greg0ire** [comment]
> I'll let you do it this time if you don't mind

I don't mind at all, I was asking exactly because of that. Will merge as soon as the build is green.

**Comment by greg0ire** [comment]
Thanks @RomainSanchez !

**Pull request merged**